src/styled.js

```diff
@@ -35,27 +35,24 @@ const styled = (Comp, config = {}) => (style = {}) => {
     const styleProps = multi ? style : { style }
     const styleKeys = Object.keys(styleProps)
 
+    // fixed styles
+    const fixedStyleProps = multi ? fixedStyle : { style: fixedStyle }
+
     for (let i = 0; i < styleKeys.length; i += 1) {
       const prop = styleKeys[i]
+      const fixedPropValue = fixedStyleProps[prop] || {}
+
       styles[prop] = {
         ...styles[prop],
         ...flatten([
           attrs[prop],
           typeof styleProps[prop] === 'function' ? styleProps[prop](props) : styleProps[prop],
           props[prop],
         ]),
+        ...fixedPropValue
       }
     }
 
-    // fixed styles
-    const fixedStyleProps = multi ? fixedStyle : { style: fixedStyle }
-    const fixedStyleKeys = Object.keys(fixedStyleProps)
-
-    for (let i = 0; i < fixedStyleKeys.length; i += 1) {
```

Comment:
You are assuming `styleKeys.length` is equal `fixedStyleKeys.length`?

Reply:
Nice catch, that is a bug, will update